clients/vault/src/main.rs

```diff
@@ -131,7 +132,7 @@ async fn start() -> Result<(), ServiceError<Error>> {
 	// This file is auto-removed when `drop`ped.
 	let _pidfile = PidFile::create(
 		&String::from(DEFAULT_SPEC_NAME),
-		signer.read().await.account_id(),
+		&AccountId::new(signer.read().await.account_id().0),
```

Comment:
Nit: It's not clear what the outcome of this statement is so let's maybe assign it to an extra variable first.